.github/workflows/build_wamr_vscode_ext.yml

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2019 Intel Corporation.  All rights reserved.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+name: build wamr-ide vscode extension
+
+on:
+  workflow_call:
+    inputs:
+      cwd:
```

Comment:
since the extension is always in `test-tools/wamr-ide/VSCode-Extension` in variant OSes, maybe we don't need it as an argument.